### ISSUE TITLE
not Starting playlist

### ISSUE THREAD

**Issue by burakguder** [ffplayout/ffplayout_engine#102]
Hi 


i get an error when i call playlist

ERROR : 

```
[2021-06-24 12:31:26,537][WARNING]  ffmpeg contains no libfdk-aac! No high quality aac...
[2021-06-24 12:31:26,537][ERROR]    ffmpeg contains no zmq filter!  Text messages will not work...
[2021-06-24 12:31:26,635][DEBUG]    Encoder CMD: "ffmpeg -v level+error -hide_banner -nostats -re -thread_queue_size 160 -i pipe:0 -metadata service_name=Live Stream -metadata service_provider=example.org -metadata year=2021 -c:v libx264 -crf 23 -x264-params keyint=50:min-keyint=25:scenecut=-1 -maxrate 1300k -bufsize 2600k -preset medium -profile:v Main -level 3.1 -c:a aac -ar 44100 -b:a 128k -flags +global_header -f flv rtmp://img.konfium.com/origin854Guo/democinema?pwd=bgm87!"
[2021-06-24 12:31:26,639][INFO]     Open: playlist.json
[2021-06-24 12:31:26,639][DEBUG]    List init
[2021-06-24 12:31:26,709][INFO]     Play: /opt/engcontent/dosya1.mp4
[2021-06-24 12:31:26,710][DEBUG]    Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -i /opt/engcontent/dosya1.mp4 -filter_complex [0:v]scale=1024:576[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -filter_complex [0:a]anull[aout1] -map [vout1] -map [aout1] -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 12:31:26,838][INFO]     Play: /opt/engcontent/dosya2.mp4
[2021-06-24 12:31:26,839][DEBUG]    Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -i /opt/engcontent/dosya2.mp4 -filter_complex [0:v]scale=1024:576[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -filter_complex [0:a]anull[aout1] -map [vout1] -map [aout1] -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 12:31:26,902][INFO]     Playlist reached end!
[2021-06-24 12:31:26,903][ERROR]    [encoder] pipe:0: Immediate exit requested
```

Playlist.json : 

```
{
    "channel": "Test 1",
    "date": "2021-06-24",
    "program": [{
            "in": 0,
            "out": 647.68,
            "duration": 647.68,
            "source": "/opt/engcontent/dosya1.mp4"
        }, {
            "in": 0,
            "out": 149,
            "duration": 149,
            "source": "/opt/engcontent/dosya2.mp4"
        }
    ]
}

```


ffplayout.yml


```
general:
    helptext: Sometimes it can happen, that a file is corrupt but still playable,
        this can produce an streaming error over all following files. The only way
        in this case is, to stop ffplayout and start it again. Here we only say when
        it stops, the starting process is in your hand. Best way is a systemd serivce
        on linux. 'stop_threshold' stop ffplayout, if it is async in time above this
        value.
    stop_threshold: 11

mail:
    helptext: Send error messages to email address, like missing playlist; unvalid
        json format; missing clip path. Leave recipient blank, if you don't need this.
        'mail_level' can be WARNING or ERROR.
    subject: "Playout Error"
    smpt_server: "mail.example.org"
    smpt_port: 587
    sender_addr: "ffplayout@example.org"
    sender_pass: "abc123"
    recipient:
    mail_level: "ERROR"

logging:
    helptext: Logging to file, if 'log_to_file' False log to console. 'backup_count'
        says how long log files will be saved in days. Path to /var/log/ only if you
        run this program as deamon. 'log_level' can be DEBUG, INFO, WARNING,
        ERROR. 'ffmpeg_level' can be INFO, WARNING, ERROR.
    log_to_file: True
    backup_count: 7
    log_path: "/var/log/ffplayout/"
    log_level: "DEBUG"
    ffmpeg_level: "ERROR"

processing:
    helptext: Default processing, for all clips that they get prepared in that way,
        so the output is unique. 'aspect' must be a float number. 'logo' is only used
        if the path exist. 'logo_scale' scale the logo to target size, leave it blank
        when no scaling is needed, format is 'number:number', for example '100:-1'
        for proportional scaling. With 'logo_opacity' logo can become transparent.
        With 'logo_filter' 'overlay=W-w-12:12' you can modify the logo position.
        With 'use_loudnorm' you can activate single pass EBU R128 loudness normalization.
        'loud_*' can adjust the loudnorm filter. 'output_count' sets the outputs for
        the filtering, > 1 gives the option to use the same filters for multiple outputs.
        This outputs can be taken in 'ffmpeg_param', names will be vout2, vout3;
        aout2, aout2 etc.
    width: 1024
    height: 576
    aspect: 1.778
    fps: 25
    add_logo: True
    logo: "docs/logo.png"
    logo_scale:
    logo_opacity: 0.7
    logo_filter: "overlay=W-w-12:12"
    add_loudnorm: False
    loud_I: -18
    loud_TP: -1.5
    loud_LRA: 11
    output_count: 1

playlist:
    helptext: Set 'playlist_mode' to 'False' if you want to play clips from the 'storage'
        section. Put only the root path here, for example '/playlists' subfolders
        are readed by the script. Subfolders needs this structur '/playlists/2018/01'
        (/playlists/year/month). 'day_start' means at which time the playlist should
        start, leave day_start blank when playlist should always start at the begin.
        'length' represent the target length from playlist, when is blank real length
        will not consider.
    playlist_mode: True
    path: "/playlists"
    day_start: "5:59:25"
    length: "24:00:00"

storage:
    helptext: Play ordered or ramdomly files from path. 'filler_clip' is for fill
        the end to reach 24 hours, it will loop when is necessary. 'extensions' search
        only files with this extension. Set 'shuffle' to 'True' to pick files randomly.
    path: "/mediaStorage"
    filler_clip: "/mediaStorage/filler/filler.mp4"
    extensions:
        - ".mp4"
        - ".mkv"
    shuffle: True

text:
    helptext: Overlay text in combination with libzmq for remote text manipulation.
        On windows fontfile path need to be like this 'C\:/WINDOWS/fonts/DejaVuSans.ttf'.
        In a standard environment the filter drawtext node is Parsed_drawtext_2.
        'over_pre' if True text will be overlay in pre processing. Continue same text
        over multiple files is in that mode not possible. 'text_from_filename' activate the
        extraction from text of a filename. With 'style' you can define the drawtext
        parameters like position, color, etc. Post Text over API will override this.
        With 'regex' you can format file names, to get a title from it.
    add_text: False
    over_pre: False
    bind_address: "127.0.0.1:5555"
    fontfile: "/usr/share/fonts/truetype/dejavu/DejaVuSans.ttf"
    text_from_filename: False
    style: "x=(w-tw)/2:y=(h-line_h)*0.9:fontsize=24:fontcolor=#ffffff:box=1:boxcolor=#000000:boxborderw=4"
    regex: "^(.*)_"


out:
    helptext: The final playout compression. Set the settings to your needs.
        'mode' has the standard options 'desktop', 'hls', 'stream'. Self made outputs
        can be define, by adding script in output folder with an 'output' function inside.
        'stream_output' is for streaming output, two ffmpeg instances are fired up, for
        pre- and post-processing. 'hls_output' is for direct output to hls playlist,
        without pre- and post-processing, mode must be 'hls'.
    mode: 'stream'
    service_name: "Live Stream"
    service_provider: "example.org"
    ffmpeg_param: >-
        -c:v libx264
        -crf 23
        -x264-params keyint=50:min-keyint=25:scenecut=-1
        -maxrate 1300k
        -bufsize 2600k
        -preset medium
        -profile:v Main
        -level 3.1
        -c:a aac
        -ar 44100
        -b:a 128k
    stream_output: >-
        -flags +global_header
        -f flv rtmp://site.com/app/stream?pwd=bopljku
    hls_output: >-
        -flags +cgop
        -f hls
        -hls_time 6
        -hls_list_size 600
        -hls_flags append_list+delete_segments+omit_endlist+program_date_time
        -hls_segment_filename /var/www/srs/live/stream-%09d.ts /var/www/srs/live/stream.m3u8
                                                                                                                                                                                                                            132,26        Bot


```
FFMPEG : 

```
ffmpeg version 4.2.4-1ubuntu0.1bbb2~18.04 Copyright (c) 2000-2020 the FFmpeg developers
  built with gcc 7 (Ubuntu 7.5.0-3ubuntu1~18.04)
  configuration: --prefix=/usr --extra-version='1ubuntu0.1bbb2~18.04' --toolchain=hardened --libdir=/usr/lib/x86_64-linux-gnu --incdir=/usr/include/x86_64-linux-gnu --arch=amd64 --enable-gpl --disable-stripping --enable-avresample --disable-filter=resample --enable-gnutls --enable-libfontconfig --enable-libfreetype --enable-libfribidi --enable-libmp3lame --enable-libopus --enable-librsvg --enable-librubberband --enable-libspeex --enable-libvorbis --enable-libvpx --enable-libxml2 --enable-sdl2 --disable-decoders --enable-decoder='aac,ass,bmp,ffv1,ffvhuff,flac,flashsv,flashsv2,flv,gif,h264,huffyuv,libopus,librsvg,libspeex,mjpeg,mp3float,mpeg2video,opus,pcm_f32le,pcm_s16le,pcm_s16le_planar,png,rawvideo,srt,ssa,subrip,vorbis,vp6,vp6a,vp6f,vp8,vp9,webp,webvtt' --disable-encoders --enable-encoder='aac,ass,bmp,ffv1,ffvhuff,flac,flashsv,flashsv2,flv,gif,h264_vaapi,huffyuv,libmp3lame,libopus,libspeex,libvorbis,libvpx_vp8,libvpx_vp9,libx264,libx264rgb,mjpeg,mjpeg_vaapi,mpeg2_vaapi,mpeg2video,pcm_f32le,pcm_s16le,pcm_s16le_planar,png,rawvideo,srt,ssa,subrip,vp8_vaapi,vp9_vaapi,webvtt' --disable-hwaccels --enable-hwaccel='h264_vaapi,h264_vdpau,mjpeg_vaapi,mpeg2_vaapi,mpeg2_vdpau,vp8_vaapi,vp9_vaapi' --disable-parsers --enable-parser='aac,bmp,flac,gif,h264,mjpeg,mpegaudio,mpegvideo,opus,png,vorbis,vp8,vp9' --disable-demuxers --enable-demuxer='aac,alsa,ass,avi,bmp_pipe,caf,concat,dash,ffmetadata,flac,flv,gif,gif_pipe,h264,hls,image2,image2pipe,ivf,jpeg_pipe,kmsgrab,lavfi,matroska,mjpeg,mov,mp3,mpegts,mpegtsraw,nut,ogg,pcm_f32le,pcm_s16le,png_pipe,pulse,rawvideo,rtp,rtsp,srt,swf,w64,wav,webvtt,yuv4mpegpipe' --disable-muxers --enable-muxer='adts,ass,avi,caf,dash,ffmetadata,flac,flv,framecrc,framehash,framemd5,gif,h264,hash,hls,image2,image2pipe,ipod,ivf,matroska,matroska_audio,md5,mjpeg,mkvtimestamp_v2,mov,mp3,mp4,mpeg2video,mpegts,nut,oga,ogg,ogv,opus,pcm_f32le,pcm_s16le,rawvideo,rtp,rtp_mpegts,rtsp,segment,singlejpeg,srt,stream_segment,swf,tee,uncodedframecrc,w64,wav,webm,webm_chunk,webm_dash_manifest,webvtt,yuv4mpegpipe' --disable-protocols --enable-protocol='async,cache,concat,data,file,hls,http,httpproxy,https,icecast,md5,pipe,rtmp,rtmps,rtmpt,rtmpts,rtp,subfile,tcp,tee,tls,udp,unix' --disable-bsfs --enable-bsf='aac_adtstoasc,chomp,filter_units,h264_metadata,h264_mp4toannexb,h264_redundant_pps,mjpeg2jpeg,mpeg2_metadata,noise,null,vp9_metadata,vp9_raw_reorder,vp9_superframe,vp9_superframe_split' --disable-indevs --enable-indev='alsa,lavfi,v4l2,fbdev,xcbgrab,kmsgrab' --disable-outdevs --enable-outdev='alsa,sdl2,xv,fbdev,v4l2' --disable-libdc1394 --enable-libdrm --disable-libiec61883 --disable-chromaprint --disable-frei0r --enable-libx264 --enable-shared
  libavutil      56. 31.100 / 56. 31.100
  libavcodec     58. 54.100 / 58. 54.100
  libavformat    58. 29.100 / 58. 29.100
  libavdevice    58.  8.100 / 58.  8.100
  libavfilter     7. 57.100 /  7. 57.100
  libavresample   4.  0.  0 /  4.  0.  0
  libswscale      5.  5.100 /  5.  5.100
  libswresample   3.  5.100 /  3.  5.100
  libpostproc    55.  5.100 / 55.  5.100
Hyper fast Audio and Video encoder

```

**Comment by jb-alvarado** [comment]
Hi, can you also post the encoder.log and decoder.log?

**Comment by burakguder** [comment]
Emcoder.log
[2021-06-24 12:57:31,160]  pipe:0: Immediate exit requeste

Decoder.log 
İs null


ffplayout.log

```
[2021-06-24 11:30:43,179] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,214] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,214] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,215] [DEBUG]  delta: -0.000033
[2021-06-24 11:30:43,215] [DEBUG]  total_delta: 66521.784968
[2021-06-24 11:30:43,215] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,215] [WARNING]  No filler is set!
[2021-06-24 11:30:43,215] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,215] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,247] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,248] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,248] [DEBUG]  delta: -0.000031
[2021-06-24 11:30:43,248] [DEBUG]  total_delta: 66521.751719
[2021-06-24 11:30:43,248] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,248] [WARNING]  No filler is set!
[2021-06-24 11:30:43,248] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,248] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,284] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,285] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,285] [DEBUG]  delta: -0.000033
[2021-06-24 11:30:43,285] [DEBUG]  total_delta: 66521.714564
[2021-06-24 11:30:43,285] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,285] [WARNING]  No filler is set!
[2021-06-24 11:30:43,285] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,286] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,317] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,318] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,318] [DEBUG]  delta: -0.000031
[2021-06-24 11:30:43,318] [DEBUG]  total_delta: 66521.681614
[2021-06-24 11:30:43,318] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,318] [WARNING]  No filler is set!
[2021-06-24 11:30:43,318] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,319] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,352] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,352] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,352] [DEBUG]  delta: -0.000032
[2021-06-24 11:30:43,352] [DEBUG]  total_delta: 66521.647238
[2021-06-24 11:30:43,353] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,353] [WARNING]  No filler is set!
[2021-06-24 11:30:43,353] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,353] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,388] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,389] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,389] [DEBUG]  delta: -0.000036
[2021-06-24 11:30:43,389] [DEBUG]  total_delta: 66521.610750
[2021-06-24 11:30:43,389] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,389] [WARNING]  No filler is set!
[2021-06-24 11:30:43,389] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,389] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"
[2021-06-24 11:30:43,424] [ERROR]  Playlist not exists: /playlists/2021/06/2021-06-24.json
[2021-06-24 11:30:43,424] [ERROR]  Clip nodes are empty!
[2021-06-24 11:30:43,424] [DEBUG]  delta: -0.000031
[2021-06-24 11:30:43,425] [DEBUG]  total_delta: 66521.575097
[2021-06-24 11:30:43,425] [ERROR]  File not exist: /mediaStorage/filler/filler.mp4
[2021-06-24 11:30:43,425] [WARNING]  No filler is set!
[2021-06-24 11:30:43,425] [INFO]  Play: color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p
[2021-06-24 11:30:43,425] [DEBUG]  Decoder CMD: "ffmpeg -v level+error -hide_banner -nostats -f lavfi -i color=c=#121212:s=1024x576:d=30:r=25,format=pix_fmts=yuv420p -f lavfi -i anoisesrc=d=30:c=pink:r=48000:a=0.05 -filter_complex [0:v]null[v];movie=docs/logo.png,loop=loop=-1:size=1:start=0,format=rgba,colorchannelmixer=aa=0.7[l];[v][l]overlay=W-w-12:12:shortest=1[vout1] -map [vout1] -map 1:a -pix_fmt yuv420p -r 25 -c:v mpeg2video -intra -b:v 58982.4k -minrate 58982.4k -maxrate 58982.4k -bufsize 29491.2k -c:a s302m -strict -2 -ar 48000 -ac 2 -f mpegts -"

```


I want the playlist I created for testing to be run.

**Comment by burakguder** [comment]
I am running the application with the following code

./ffplayout.py -l none -p playlist.json -s now -t none -m stream

**Comment by jb-alvarado** [comment]
Hm, to me everything looks normal, except that everything rans way to fast. Can you try one time a self compiled version from ffmpeg? You can use this script for it: https://github.com/jb-alvarado/compile-ffmpeg-osx-linux

**Comment by burakguder** [comment]
unfortunately i can't compile the code


```
HEAD is now at 947122f libavfilter: Fix fate-source after 072788c46e36a21ca9e8f1e3cc19a1944db5b89c
From https://github.com/FFmpeg/FFmpeg
 * branch            master     -> FETCH_HEAD
Already up to date.
install prefix            /root/playout/ffplayout_engine/compile-ffmpeg-osx-linux/local

```

.......
.......
......
```

libavdevice/avdevice.c: In function ‘avdevice_configuration’:
./config.h:4:30: error: missing terminating " character
 #define FFMPEG_CONFIGURATION "--disable-debug --disable-shared --disable-doc --enable-gpl --enable-version3 --enable-runtime-cpudetect --enable-avfilter --enable-zlib --enable-pic
                              ^
./config.h:4:30: note: in definition of macro ‘FFMPEG_CONFIGURATION’
 #define FFMPEG_CONFIGURATION "--disable-debug --disable-shared --disable-doc --enable-gpl --enable-version3 --enable-runtime-cpudetect --enable-avfilter --enable-zlib --enable-pic
                              ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
libavdevice/avdevice.c:44:5: warning: ‘return’ with no value, in function returning non-void
     return FFMPEG_CONFIGURATION;
     ^~~~~~
libavdevice/avdevice.c:42:14: note: declared here
 const char * avdevice_configuration(void)
              ^~~~~~~~~~~~~~~~~~~~~~
ffbuild/common.mak:70: recipe for target 'libavdevice/avdevice.o' failed
make: *** [libavdevice/avdevice.o] Error 1
-------------------------------------------------
build ffmpeg-git failed...
delete the source folder under '/root/playout/ffplayout_engine/compile-ffmpeg-osx-linux/build' and start again,
or if you know there is no dependences hit enter for continue it
first close the batch window, then the shell window
```

**Comment by burakguder** [comment]
I installed ffmpeg 4.3.2, problem solved

For ubuntu you can follow the below steps

```
sudo apt install software-properties-common
sudo apt update
sudo add-apt-repository ppa:jonathonf/ffmpeg-4
sudo apt install ffmpeg
ffmpeg -version
```
